src/Sulu/Component/Content/Document/Query/Builder/OrderByAdd.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Query\Builder;
+
+/**
+ * @see Doctrine\ODM\PHPCR\Query\Builder\OrderByAdd
+ */
+class OrderByAdd extends OrderBy
+{
+}
```

Comment:
Yes. These are necessary, you will find an explanation in a comment in
the code.

On Fri, Feb 12, 2016 at 04:46:45AM -0800, Daniel Rotter wrote:
>    In [1]src/Sulu/Component/Content/Document/Query/Builder/OrderByAdd.php:
> 
>  > + * This file is part of Sulu.
>  > + *
>  > + * (c) MASSIVE ART WebServices GmbH
>  > + *
>  > + * This source file is subject to the MIT license that is bundled
>  > + * with this source code in the file LICENSE.
>  > + */
>  > +
>  > +namespace Sulu\Component\Content\Document\Query\Builder;
>  > +
>  > +/**
>  > + * @see Doctrine\ODM\PHPCR\Query\Builder\OrderByAdd
>  > + */
>  > +class OrderByAdd extends OrderBy
>  > +{
>  > +}
> 
>    Does this class serve a purpose?
> 
>    —
>    Reply to this email directly or [2]view it on GitHub.
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu-io/sulu/pull/1863#discussion_r52736948
>    2. https://github.com/sulu-io/sulu/pull/1863/files#r52736948
>    3. https://github.com/sulu-io/sulu/pull/1863/files#r52736948